build/crd-samples/sedna/lifelonglearningjobv1alpha1.yaml

```diff
@@ -0,0 +1,79 @@
+apiVersion: sedna.io/v1alpha1
+kind: LifelongLearningJob
+metadata:
+  name: atcii-classifier-demo
+spec:
+  dataset:
+    name: "lifelong-dataset"
+    trainProb: 0.8
+  trainSpec:
+    template:
+      spec:
+        nodeName: $WORKER_NODE
```

Comment:
this is example yaml, the nodeName try to be more realistic, and `$WORKER_NODE` seems not realistic, so change to another good value.